website/content/docs/enterprise/license/utilization-reporting.mdx

```diff
@@ -0,0 +1,190 @@
+metering data to HashiCorp without requiring you to manually collect and report
+them. It also lets you review your entitlement usage with the monitoring
+solution you already use (Splunk, Datadog, etc.) so you can optimize and manage
+your deployments. 
+
+Utilization reporting shares the minimum data required to validate entitlement
+utilization as contractually obligated. They consist of mostly computed metrics
+and will never contain PII or other sensitive information. Utilization reporting
+shares the data with HashiCorp using a secure, unidirectional HTTPS API, and
+makes an auditable record in the product logs each time it submits a report.
+
+To enable automated reporting you need to upgrade to a version that supports it,
+and open your network to outbound HTTPS traffic on port 443 to
+`https://reporting.hashicorp.services`. If your installation is completely
+air-gapped, you can’t use automated entitlement utilization and can opt out
+instead. You do not need to re-sign the EULA to opt in or out. 
+
+## Enable reporting
+
+To enable automated entitlement utilization reporting, upgrade to a version that
+supports it and make sure that your network allows the right egress traffic. 
+
+### 1. Allow outbound traffic on port 443
+
+Make sure that your network allows HTTPS egress on port 443 to
+https://reporting.hashicorp.services. 
+
+### 2. Upgrade
+
+Upgrade to a release that supports entitlement utilization reporting. These
+releases include:
+
+- [Vault Enterprise 1.14.0](https://releases.hashicorp.com/vault/) and later
+
+
+### 3. Check logs
+
+Automatic entitlement utilization reporting will start sending data within 24
+hours. Check the product logs for records that the data sent successfully. 
+
+Instructions on how to check logs. 
+
+<CodeBlockConfig hideClipboard>
+
+```
+[DEBUG] beginning snapshot export
+[DEBUG] creating payload
+[DEBUG] marshalling payload to json
+[DEBUG] generating authentication headers
+[DEBUG] creating request
+[DEBUG] sending request
+[DEBUG] performing request: method=POST url=https://census.license.hashicorp.services
+[DEBUG] recording audit record
+[INFO]  {"payload":{"payload_version":"1","license_id":"974b62bd-7e7b-4556-7741-9683ecd9acb4","product":"vault","product_version":"1.2.3","export_timestamp":"2023-04-14T15:06:35.258709-04:00","snapshots":[{"snapshot_version":1,"snapshot_id":"0001J3K91VB1SYJGSHM9JTFXJ8","process_id":"01GY0JJYXZ6MY01GNM9VN88ZEJ","timestamp":"2023-04-14T15:06:35.257143-04:00","schema_version":"0.1.0","service":"vault","metrics":{"clientcount.current_month_estimate":{"key":"clientcount.current_month_estimate","kind":"sum","mode":"write","value":117,"labels":{"type":{"entity":58,"nonentity":59}}},"clientcount.previous_month_complete":{"key":"clientcount.previous_month_complete","kind":"sum","mode":"write","value":300,"labels":{"type":{"entity":100,"nonentity":200}}}}}],"metadata":{"vault":{"billing_start":"2023-03-24T00:00:00Z"}}}}
```

Comment:
It looks like this output was updated to include the "Report sent" prefix. We also use a named logger for this, so:
```suggestion
[INFO]  reporting: Report sent: auditRecord={"payload":{"payload_version":"1","license_id":"974b62bd-7e7b-4556-7741-9683ecd9acb4","product":"vault","product_version":"1.2.3","export_timestamp":"2023-04-14T15:06:35.258709-04:00","snapshots":[{"snapshot_version":1,"snapshot_id":"0001J3K91VB1SYJGSHM9JTFXJ8","process_id":"01GY0JJYXZ6MY01GNM9VN88ZEJ","timestamp":"2023-04-14T15:06:35.257143-04:00","schema_version":"0.1.0","service":"vault","metrics":{"clientcount.current_month_estimate":{"key":"clientcount.current_month_estimate","kind":"sum","mode":"write","value":117,"labels":{"type":{"entity":58,"nonentity":59}}},"clientcount.previous_month_complete":{"key":"clientcount.previous_month_complete","kind":"sum","mode":"write","value":300,"labels":{"type":{"entity":100,"nonentity":200}}}}}],"metadata":{"vault":{"billing_start":"2023-03-24T00:00:00Z"}}}}
```